member/resource.py

```diff
@@ -21,5 +21,23 @@ class Member(MethodResource):
     def get(self, id):
         return Model.query.get_or_404(id)
 
+    @doc(
+        summary="Modify a particular member",
```

Comment:
Can you make the wording fit with the rest of the summaries. I specifically went to remove all mentions of "particular" from other entries (probably after you made this)

Reply:
yeah it was after I think